### DISCUSSION TITLE
Trivia game on Thursday!

### DISCUSSION THREAD

**Discussion by olzama** [delph-in/docs#51] [Team Posts]
Dear @delph-in/participants ,

We'd like to gather some fun or less-known facts about DELPH-IN Participants for a trivia-type game at our upcoming social on July 22. We hope this will be a fun way for us to reconnect or for new people to get to know people a little better! During the game, we will be guessing which fact is about which person :).

We are looking for things like "fun facts" i.e., a hobby or past experience, or anything, really. For example, maybe you play an instrument, had a crazy hairstyle in high school, or belong to a club, or collect something. We realize DELPH-IN goes back a long way so many people will know many facts about each other but we are sure this will still be fun. (P.S.: We hope everyone will attend but please send us something even if you do not plan to attend.)

**Please fill out the form:**
 
https://forms.gle/boJdUgH6AiGptUsu7

Or, just send Olga your answers via email! 

Don't post your answers here though, as that will defeat the purpose of the game :).

Please do this **today or tomorrow**, in order for me to be able to put the game together :).

**Comment by olzama** [comment]
Dear @delph-in/participants ,

We are looking forward to seeing all of you at tomorrow's Social which will be the entire Session 1 (via Zoom). It is meant to "replace" the traditional DELPH-IN social half-day which is normally followed by a banquet.

**One important thing:** Try to have your **phone** (a smartphone) near you, as it will be a lot more convenient for you to participate via the phone while having your computer displaying the Zoom session. We will spend some time at the beginning on setting things up (so don't be too late :) ).

Also, if you haven't yet sent me a "fun fact" about yourself, please do!

**Comment by olzama** [comment]
Please don't forget also to use headphones; this way we will get fewer echo problems and may be able to spend more time unmuted! Which is the goal.

**Comment by olzama** [comment]
Thanks everyone for participating! 

<img width="1917" alt="Screen Shot 2021-07-22 at 8 02 14 AM" src="https://user-images.githubusercontent.com/10963114/126662802-8432c3e1-f0a0-4159-8259-a0d7266ff27d.png">

**Comment by olzama** [comment]
And here's just a couple more facts which we didn't get to: 

<img width="1133" alt="Screen Shot 2021-07-22 at 8 09 53 AM" src="https://user-images.githubusercontent.com/10963114/126662966-71efe73b-f0f2-4771-889d-b271ae656746.png">

<img width="1141" alt="Screen Shot 2021-07-22 at 8 10 02 AM" src="https://user-images.githubusercontent.com/10963114/126662985-191ff422-0afd-469f-b996-fb5f01630a90.png">

**Comment by arademaker** [comment]
Second is Francis

**Comment by olzama** [comment]
> Second is Francis

Yes, this one was supposed to be the very last (and very easy for people to guess, and I had a prepared joke about how nobody is going to get THIS right!)

**Comment by olzama** [comment]
During the game, we quickly realized that many of the Fun Facts really call for photographic (or in some cases, video) evidence! We would really love to see @lascarid  with her hair braided together with pieces of torn pillowcases; the beautiful chrysanthemum for which @john-a-carroll  won 1st prize at a local competition; @EricZinda 's giant clam; @allisondods 's favorite lamp photo; and perhaps a photo of @danflick  when he was 5 years old :). That's just to name a few (and we already saw @arademaker 's tennis rackets 🤣). As for @curtosis  who couldn't make it, we want to hear everything about his time at gold mines! :) Possibly also with pictures? :) This discussion thread is private; perhaps people could just add their pictures here as they come across them?

**Comment by olzama** [comment]
By the way, in my "things that really annoy you" Facebook group today: Someone complaining about their vacuum cleaner (the Roomba type; that it often gets confused, etc.) --- followed by a delightful, lengthy thread of comments where people (who also own similar vacuum cleaners and also are annoyed by them) quote what exactly they tend to say to the devices ("Hey! What are you doing under there?! Give me THAT!!"). Apparently, a robotic vacuum cleaner is often seen as a suitable interlocutor. (To those who didn't attend: my own "fun fact" was that I am a member of a Facebook group in which people share things that really annoy them.)

**Comment by emilymbender** [comment]
Okay, photographic evidence adjacent to my "live and work within 2 miles (3.2km) of where I was born" fun fact:

![20210722_185412](https://user-images.githubusercontent.com/12276474/126729568-18853b64-35bf-402b-aa68-c74a8d60d48a.jpg)

That's from the Seattle Times on a slow news day when I was a toddler... photo taken at the Woodland Park Zoo (about 3 miles/5km from UW Hospital).

**Comment by glenn-slayden** [comment]
Cali, Colombia, May 1972
![33565F06-178F-432D-BD2C-43C3BB303855](https://user-images.githubusercontent.com/5589855/126742123-475b0d7e-5858-4898-908d-00d2f471b06c.jpeg)

**Comment by john-a-carroll** [comment]
My prize-winning chrysanthemum. It's an 'incurve' variety bred for showing, where the aim is to get a perfectly balanced globe with the petals coming together at the top without any gap.

![chyrsanthemum-prize](https://user-images.githubusercontent.com/31476017/126753786-94419292-f02e-4b03-8067-416dac6873ac.jpg)

(This is from a print of a colour photo taken in the days before digital, which is now badly faded - so I converted to B&W which looks a bit better.)

**Comment by lascarid** [comment]
Wow!  That is one beautiful flower!

**Comment by lascarid** [comment]
Like I said yesterday, I'm currently in London and my pillowcase hair do was so long ago (1983) I only have a physical photo.  I'll scan and post here when I return to Edinburgh next week.

**Comment by emilymbender** [comment]
Beautiful flower, John!!

**Comment by emilymbender** [comment]
@lascarid we'll be looking for that photo!!!

**Comment by lizcconrad** [comment]
Me just moments before disaster... (but many moments after the first disaster 🦢💩)
![Screen Shot 2021-07-22 at 9 00 27 AM](https://user-images.githubusercontent.com/17073817/126815047-4328ead6-129e-41a1-9b56-7333ab568766.png)

**Comment by emilymbender** [comment]
@lizcconrad you look entirely unsure whether the cake is sufficient consolation....

**Comment by EricZinda** [comment]
Unfortunately I couldn't find an actual photo of the clam I raised, but I did find an example of one that shows a scuba diver for size reference!
![image](https://user-images.githubusercontent.com/2416921/126816455-a920303a-b28b-4a0a-a79f-4e463bfb8a18.png)

**Comment by EricZinda** [comment]
Mine only got about 6-7 inches before it outgrew the tank, unfortunately!

**Comment by EricZinda** [comment]
I did find a picture of my little beauty! Not the best photo, but...
![Clam](https://user-images.githubusercontent.com/2416921/126817428-e27c8b1b-d060-42d3-953a-7cfe06e8dec0.jpg)

**Comment by allisondods** [comment]
I unfortunately don't have any photos of my own saved anymore from this
trip, but the lampposts that kickstarted my love of lighting appliances can
be seen in this Pinterest photo
<https://www.pinterest.com/pin/254383078929158575/>, at Gadjah Mada
University in Indonesia.

My favorite from my own collection are these lampposts in Seattle's
Volunteer Park:

[image: AE608A88-A5A8-476A-80D3-35295147018D_1_105_c.jpeg]

**Comment by olzama** [comment]
Hmm somehow @allisondods 's picture is not displaying for me... Could you try editing your post and dropping the picture once more?

**Comment by lascarid** [comment]
Sorry these aren't great quality photos...The guy in the one with me in pillowcases is Colin, the hairdresser who used me as his project.  The photo of me with big hair is what I looked like when my cousin Markella (in the background of the photo) and I both worked our fingers to the bone for 8 hours, unplaitting and patiently combing out the dreadlocks.

![solo](https://user-images.githubusercontent.com/40396759/126909786-29ca90a2-81b0-4daf-8e14-5e3f0038be03.png)

![alex_and_colin](https://user-images.githubusercontent.com/40396759/126909781-83a71f7c-0327-4b7a-a3e8-4ea166f3f676.png)

![after](https://user-images.githubusercontent.com/40396759/126909773-b67a270f-74c4-4acb-ba9a-85cdf5c44fdb.png)

**Comment by olzama** [comment]
These are fantastic, @lascarid , thank you for sharing!! How old are you in these photos? You look like a baby! :)

**Comment by lascarid** [comment]
I was 20...from where I stand now that does feel like a baby!

**Comment by emilymbender** [comment]
These are amazing!! And you also totally look like yourself as I know you :)

**Comment by olzama** [comment]
@lascarid I thought you were 14 or something 🤣🤦‍♀️

**Comment by guyemerson** [comment]
![Pippi](https://user-images.githubusercontent.com/5824426/127717407-1e382955-1d41-4fd6-8c36-bed12eff7040.JPG)

My current Pippi Longstocking collection

**Comment by olzama** [comment]
@guyemerson , here's a virtual addition to your collection!
![peppi1](https://user-images.githubusercontent.com/10963114/127726045-c0da86f6-7241-4779-93a9-c6529bcabbb6.jpg)

**Comment by olzama** [comment]
Also, I see the Georgian edition has been improved by the talented previous owner (in terms of the cover).
